libgiza/config.py

```diff
@@ -140,9 +140,9 @@ def get_dict_value(v):
 
         for key, value in self.state.items():
             if safe in (True, None):
-                if key.startswith('_'):
+                if key != "_id" and key.startswith('_'):
                     continue
-                elif key in ('pass', 'password', 'token', 'key', 'secret'):
+                elif 'pass' in key or key in ('token', 'key', 'secret'):
```

Comment:
What is the value in this change? If there are other pass* words we want to match, could we instead do
    elif key.startswith('pass') or key in ('token', 'key', 'secret')
just to be more specific in what we match?